index.html

```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
       <title>Butter Component Tester</title>
+      <link  rel="stylesheet" type="text/css" href="./node_modules/butter-base-theme/index.css"/>
```

Comment:
i guess it should be named `butter-theme-default` or else it's going to be hard to make an installer